src/app/core/market/api/profile/profile.model.ts

```diff
@@ -1,6 +1,40 @@
+import { Cart } from '../cart/cart.model';
+import { Favorite } from '../favorites/favorite.model';
+
 export class Profile {
 
-    constructor(private object: any) {
+  public shippingAddresses: Array<any>;
+  public shoppingCarts: Array<Cart>;
+  public favorites: Array<Favorite>;
+
+  constructor(private object: any) {
+    this.setShippingAddresses();
+    this.setShoppingCarts();
+    this.setFavoriteItems();
+  }
+
+  get id() {
+    return this.object.id;
+  }
+
+  get address() {
+    return this.object.address;
+  }
+
+  get name() {
+    return this.object.name;
+  }
+
+  setShippingAddresses() {
+    this.shippingAddresses = this.object.ShippingAddresses
+                            .filter((address) => address.title && address.type === 'SHIPPING_OWN')
+  }
+
+  setShoppingCarts() {
+    this.shoppingCarts = this.object.ShoppingCart.filter((shop) => shop.name === 'DEFAULT')
```

Comment:
Not sure if we want these filters by default.. hmmm, need to think about it

Reply:
yeah :)